examples/data/austin-geojson.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Can we use https://github.com/uber-common/deck.gl-data instead?

Reply:
The vancouver data? It's a little large to just demonstrate multi select I think, but I could write another example using it

Reply:
Maybe you can push this data there and use it using Ajax.